manifest-generator/generator-services/src/main/java/io/hyscale/generator/services/model/ManifestGeneratorActivity.java

```diff
@@ -26,7 +26,8 @@
     INVALID_SIZE_FORMAT("Invalid size format {} "),
     INSUFFICIENT_MEMORY("Ignoring memory limits as 4Mi is minimum memory but declared is {}"),
     INSUFFICIENT_CPU("Ignoring CPU limits as 1m is minimum memory but declared is {}"),
-    INVALID_RANGE("Invalid range {}");
+    INVALID_RANGE("Invalid range {}"),
+    FAILED_TO_CREATE_IMAGE_PULL_SECRET("Missing pull credentials for the registry {}. In case of deployment failure perform 'docker login {} ' to continue with the deployment");
```

Comment:
Missing credentials for registry {}. Please perform 'docker login {}'  to continue with deployment

Reply:
Probably please might be too pleasing :P